src/lib.rs

```diff
@@ -10,6 +10,9 @@ use notify::{RecommendedWatcher, RecursiveMode, Watcher};
 use structopt::StructOpt;
 use wasm_bindgen_cli_support::Bindgen;
 
+#[cfg(feature = "wasm-opt")]
+pub mod wasm_opt;
```

Comment:
Re-export the `run` function of wasm_opt here and rename it to just `wasm_opt`. Make the module private.

This will simplify the API for the user.